doc/man/restic-backup.1

```diff
@@ -85,6 +85,10 @@ given as the arguments.
 \fB\-\-time\fP=""
     time of the backup (ex. '2012\-11\-01 22:08:41') (default: now)
 
+.PP
+\fB\-\-use\-windows\-vss\fP
+    use windows volume shadow copy service
+
```

Comment:
Unfortunately the man pages are generated when building a release, so editing this file won't have any effect. Please see a similar discussion at https://github.com/restic/restic/pull/2461#issuecomment-556301307 for more details! 👍

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/ad5f4ba0780b4940ff87cb3003c5cf159f24e91b